project/settings.py

```
@@ -27,24 +27,71 @@
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+if not SECRET_KEY:
+    raise ValueError("No DJANGO_SECRET_KEY set for Django application. Please set it in your environment variables.")
 
-ALLOWED_HOSTS = [
-    "0.0.0.0",
-    ".claude",
+# SECURITY WARNING: don't run with debug turned on in production!
+# DEBUG is True by default for development, False for production.
+# Set DJANGO_DEBUG=True in your .env file for local development if needed.
+DEBUG = os.environ.get('DJANGO_DEBUG', 'False') == 'True'
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)